packages/website/src/hooks/useScrolled.ts

```diff
@@ -1,9 +1,10 @@
 import * as React from "react";
+import { useIsomorphicEffect } from "reakit-utils/useIsomorphicEffect";
 
 export default function useScrolled(offset = 0) {
   const [scrolled, setScrolled] = React.useState(false);
 
-  React.useLayoutEffect(() => {
+  useIsomorphicEffect(() => {
     const handleScroll = () => {
       setScrolled(window.scrollY > offset);
```

Comment:
This will most likely throw an error server-side now when `window` is `undefined`. It might be a good idea to conditionally check if `window` is set here.

There are a few other instances of this in this PR as well.

Reply:
@jperasmus I don't see why it would? I'm testing the `typeof window` not `window === undefined`.

Reply:
I was under the impression that `React.useEffect()` would run on the server instead of `React.useLayoutEffect()`, but read a bit more now and saw that neither of them run server-side. `useEffect` just doesn't have the warning.

If `useEffect` also doesn't run server-side, this should be fine. 👍

Reply:
I'm sorry just now grasping that you ment when the effect is actually ran... But when rendering using SSR the `useEffect` is actually never ran, but picked up during rehydration (and on the client it'll use the useLayoutEffect instead). The behaviour is actually quite confusing as both cases end up doing the same effectively (just not in the same animation frame).